Cargo.toml

```diff
@@ -28,7 +28,6 @@ insta = { version = "1.28.0", features = ["yaml"] }
 itertools = "0.13.0"
 js-source-scopes = "0.4.0"
 lazy_static = "1.4.0"
-libfuzzer-sys = "0.4"
```

Comment:
interesting, this comes from here: https://github.com/getsentry/symbolic/blob/032b134a3f45316bf37accd29712d832557e55af/symbolic-debuginfo/fuzz/Cargo.toml#L12

might be worth running a bit of fuzzing again some day, its been years since the last time I did that :-D

Reply:
Yeah, I originally wanted to move those to workspace deps, but it doesn't work because the `-fuzz` creates live in their own workspaces.